packages/custom-templated-path-webpack-plugin/package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "@wordpress/custom-templated-path-webpack-plugin",
+	"version": "1.0.0",
```

Comment:
This initial version here needs to *not* be `1.0.0`, @gziolo has used `"version": "1.0.1-0",` in the past which has worked fine though I think we should use `1.0.0-alpha`. 

This all because Lerna _bumps_ this version upon release, if it were to stay at `1.0.0` then Lerna would only be able to publish a _patch_ `1.0.1`, _minor_ `1.1.0`, or _major_ `2.0.0` release and a `1.0.0` version would never be published.

Reply:
Well, that's annoying 😄 Maybe it'd be fine to publish an initial release manually, without Lerna?